### DISCUSSION TITLE
Unable to log in to MyGithub App using Token.(HTTP401)

### DISCUSSION THREAD

**Discussion by Skylight-666** [community/community#20399] [Projects and Issues] [answered]
![Screenshot_20220713_014932](https://user-images.githubusercontent.com/109013150/178564990-56db796c-b514-49de-9f8b-6863e206fa8b.jpg)

**Comment by FahdSeddik** [comment, marked as answer]
Hey!
Try restarting the app and force stopping it. If it still did not work try clearing cache. Hope this helps!
